external-crates/move/crates/move-vm-config/src/runtime.rs

```diff
@@ -56,24 +62,24 @@ impl Default for VMRuntimeLimitsConfig {
     }
 }
 
-#[cfg(debug_assertions)]
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
 pub struct VMProfilerConfig {
-    /// Base path for files
-    pub base_path: std::path::PathBuf,
+    /// User configured full path override
+    pub full_path: std::path::PathBuf,
     /// Whether or not to track bytecode instructions
     pub track_bytecode_instructions: bool,
     /// Whether or not to use the long name for functions
     pub use_long_function_name: bool,
 }
 
-#[cfg(debug_assertions)]
-impl std::default::Default for VMProfilerConfig {
-    fn default() -> Self {
-        Self {
-            base_path: std::path::PathBuf::from("."),
-            track_bytecode_instructions: false,
-            use_long_function_name: false,
+#[cfg(feature = "gas-profiler")]
+impl VMProfilerConfig {
+    pub fn get_default_config_if_enabled() -> Option<VMProfilerConfig> {
+        println!("tried to take the default config");
```

Comment:
Nit: extraneous println

I also believe the `PROFILER_ENABLED` check can be removed below?

Reply:
I was looking to leave it so if someone wants to profile a test they can still use the env var